.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "fastlane/actions/common"]
 	path = fastlane/actions/common
-	url = git@github.com:aol-public/OneMobileSDK-build-scripts-ios.git
+	url = https://github.com/aol-public/OneMobileSDK-build-scripts-ios.git
```

Comment:
Can we leave ssh here? Any proofs that with ssh it will never work.

Reply:
Emm, well, when I try to use ssh, Travis is simply not able to pull this submodule. We use https in player controls repo, so it shouldn't be a big trouble.

Reply:
Is it impossible to make it working through ssh?

Reply:
Maybe this can be used - https://git-scm.com/docs/git-config#git-config-urlltbasegtinsteadOf ?

Reply:
How exactly do you see that?

Reply:
You can only use `insteadOf` locally - means, that you just set in your git config, that all `https` URL's should be replaced with `git@`. But our public repos can't be fetched using ssh on Travis. Because, in our private repos, we can set an ssh to fetch our private repos. If someone will create a branch in our open-sourced repo and will push it on Travis, there will be build created, that will use that ssh, that is able to fetch private repos. Obviously it is not secure, which is why the `https` method is the only one to go.

Reply:
Ok, got it.